stackit/internal/services/logme/instance/resource.go

```diff
@@ -52,12 +54,56 @@ type Model struct {
 
 // Struct corresponding to DataSourceModel.Parameters
 type parametersModel struct {
-	SgwAcl types.String `tfsdk:"sgw_acl"`
+	SgwAcl                 types.String  `tfsdk:"sgw_acl"`
+	EnableMonitoring       types.Bool    `tfsdk:"enable_monitoring"`
+	FluentdTcp             types.Int64   `tfsdk:"fluentd_tcp"`
+	FluentdTls             types.Int64   `tfsdk:"fluentd_tls"`
+	FluentdTlsCiphers      types.String  `tfsdk:"fluentd_tls_ciphers"`
+	FluentdTlsMaxVersion   types.String  `tfsdk:"fluentd_tls_max_version"`
+	FluentdTlsMinVersion   types.String  `tfsdk:"fluentd_tls_min_version"`
+	FluentdTlsVersion      types.String  `tfsdk:"fluentd_tls_version"`
+	FluentdUdp             types.Int64   `tfsdk:"fluentd_udp"`
+	Graphite               types.String  `tfsdk:"graphite"`
+	IsmDeletionAfter       types.String  `tfsdk:"ism_deletion_after"`
+	IsmJitter              types.Float64 `tfsdk:"ism_jitter"`
+	IsmJobInterval         types.Int64   `tfsdk:"ism_job_interval"`
+	JavaHeapspace          types.Int64   `tfsdk:"java_heapspace"`
+	JavaMaxmetaspace       types.Int64   `tfsdk:"java_maxmetaspace"`
+	MaxDiskThreshold       types.Int64   `tfsdk:"max_disk_threshold"`
+	MetricsFrequency       types.Int64   `tfsdk:"metrics_frequency"`
+	MetricsPrefix          types.String  `tfsdk:"metrics_prefix"`
+	MonitoringInstanceId   types.String  `tfsdk:"monitoring_instance_id"`
+	OpensearchTlsCiphers   types.List    `tfsdk:"opensearch_tls_ciphers"`
+	OpensearchTlsProtocols types.List    `tfsdk:"opensearch_tls_protocols"`
```

Comment:
What was the conclusion regarding the inclusion of the `Groks`?

Reply:
I won't add support for that field for now, we can consider it if we get a request or once the team improves their "GetInstance" endpoint response to actually return the correct types instead of the generic object